src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -376,11 +376,11 @@ def _imaging_frames_to_hdmf_iterator(
     ----------
     imaging : ImagingExtractor
         The imaging extractor to get the data from.
-    iterator_type : {'v2', 'v1'}
-        The type of iterator to use.
+    iterator_type: {"v1", "v2", None}, default: 'v2'
```

Comment:
In numpy doc notation, the default value appears first: https://numpydoc.readthedocs.io/en/latest/format.html#parameters

Reply:
See the comments in the other PR, in this case it can be None as well:

https://github.com/catalystneuro/neuroconv/blob/3570c50a6aa2123ac43c9cbb88c4143d187a7d52/src/neuroconv/tools/spikeinterface/spikeinterface.py#L553-L555

Reply:
True, I will change that.

Reply:
Done in the last commit.